account_commission/views/report_settlement_templates.xml

```diff
@@ -38,11 +38,5 @@
                 </td>
             </t>
         </xpath>
-        <xpath expr="//table/tfoot/tr/td" position="before">
```

Comment:
Why removing this?

Reply:
this inheritance add extra cells to be able to align the total on right whatever the content.
Yet AFAICS, using tfoot repeat the footer on each page so using tfoot looks inefficient to me.
And when using a dedicated table for the total summary, no need to use extra cell to be aligned.

Reply:
But is it used in any dependent module? If so, it should be changed there as well.

Reply:
actually, this inheritance in the account module is done because of the structure of the main view.
Depending of the module, it adds extra columns. That's why account_commission was adding a new column.
Now, with the simple syntax outside the table, there is no more problems for displaying properly the total, whatever the module adding new columns.